olclient/helper_classes/results.py

```diff
@@ -0,0 +1,105 @@
+from typing import Optional
+
+from olclient.common import Book
+from olclient.utils import extract_olid_from_url
+
+
+class Results:
+    """Container for the results of the Search API"""
+
+    def __init__(self, start=0, num_found=0, docs=None, **kwargs):
+        self.start = start
+        self.num_found = num_found
+        self.docs = [self.Document(**doc) for doc in docs] or []
+
+    @property
+    def first(self):
+        if self.docs:
+            return self.docs[0]
+
+    class Document:
+        """An aggregate OpenLibrary Work summarizing all Editions of a Book"""
+
+        def __init__(
+            self,
+            key,
+            title: str = "",
+            subtitle: Optional[str] = None,
```

Comment:
```suggestion
            subtitle: str = "",
```
?

Reply:
The type in the method documentation says it should be a `subtitle (unicode) [optional]` (https://github.com/internetarchive/openlibrary-client/blob/731dbe57373de63a6b4a4a9776ffaedc87f23f40/olclient/openlibrary.py#L1167). In that case, `Optional[str]` is correct, right?